lib/ansible/plugins/__init__.py

```diff
@@ -137,23 +134,48 @@ def _check_required(self):
         # FIXME: standardize required check based on config
         pass
 
+    def __repr__(self):
+        load_name = getattr(self, '_load_name', '(unknown)')
```

Comment:
shouldn't we use `ansible_name` instead of `_load_name` now?

Reply:
This function now includes both the fully-qualified name and the load name.